lib/execution_engine2/sdk/EE2StatusRange.py

```diff
@@ -75,8 +75,10 @@ def check_jobs_date_range_for_user(
         if offset is None:
             offset = 0
 
+        if user is None:
+            user = self.sdkmr.get_user_id()
         # Admins can view "ALL" or check_jobs for other users
-        if user != self.sdkmr.user_id:
+        if user != self.sdkmr.get_user_id():
```

Comment:
is there a nicer way to do this so you don't set `user = self.sdkmr.get_user_id()` and then immediately check whether user is `self.sdkmr.get_user_id()`?

Reply:
yes